jwst/source_catalog/source_catalog.py

```diff
@@ -111,8 +111,8 @@ def convert_to_jy(self):
         `~astropy.unit.Quantity` objects.
         """
         in_unit = 'MJy/sr'
-        if (self.model.meta.bunit_data != in_unit
-                or self.model.meta.bunit_err != in_unit):
+        if (self.model.meta.bunit_data != in_unit or
+                self.model.meta.bunit_err != in_unit):
```

Comment:
Why was this changed?  The PEP8 standard changed in 2016 such that all new code should have the line break before operators.  https://www.flake8rules.com/rules/W504.html

The W503 warning is outdated (and conflicts with W504).

Reply:
At https://peps.python.org/pep-0008/ in the section on conditionals it appears the `or` should go at the end of the preceding line, instead of the beginning of the next line.